apps/music/test/marionette/lib/listviewhelper.js

```diff
@@ -0,0 +1,18 @@
+/* global module, require */
+'use strict';
+
+var assert = require('assert');
+
+var ListviewHelper = {
+
+  backgroundImageForListItem: function(list_items, index) {
```

Comment:
I'd change the function name. If the implementation changes to use an <img> tag instead of background-image, then this name wouldn't make sense. albumArtForListItem() would make sense.  (Of course, then there is the issue of whether you should remove the "url()" from the returned string or not...)

The other helper classes I've seen take a single element as their argument. Here you're using an array and index which seems non-parallel. I'd modify to just pass the element and do the indexing in the code that calls this function.